validator/client/src/main/java/tech/pegasys/teku/validator/client/OwnedKeyManager.java

```diff
@@ -44,41 +44,36 @@
 import tech.pegasys.teku.validator.client.restapi.apis.schema.ImportStatus;
 import tech.pegasys.teku.validator.client.restapi.apis.schema.PostKeyResult;
 
-public class ActiveKeyManager implements KeyManager {
+public class OwnedKeyManager implements KeyManager {
   private static final String EXPORT_FAILED =
       "{\"metadata\":{\"interchange_format_version\":\"5\"},\"data\":[]}";
   private static final Logger LOG = LogManager.getLogger();
   private final ValidatorLoader validatorLoader;
   private final ValidatorTimingChannel validatorTimingChannel;
 
-  public ActiveKeyManager(
+  public OwnedKeyManager(
       final ValidatorLoader validatorLoader, final ValidatorTimingChannel validatorTimingChannel) {
     this.validatorLoader = validatorLoader;
     this.validatorTimingChannel = validatorTimingChannel;
   }
 
-  /**
-   * Get a listing of active validator keys
-   *
```

Comment:
Just this comment needs changing I think

Reply:
We agreed that just removing it is ok